lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php

```diff
@@ -76,10 +72,7 @@ function anything()
  */
 function arrayHasKey($key)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$key". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)